radarr/movieeditor.go

```diff
@@ -0,0 +1,69 @@
+package radarr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"golift.io/starr"
+)
+
+const bpMovieEditor = bpMovie + "/editor"
+
+// BulkEdit is the input for the bulk movie editor endpoint.
+// You may use starr.True(), starr.False(), starr.Int64(), and starr.String() to add data to the struct members.
+type BulkEdit struct {
+	MovieIDs            []int64 `json:"movieIds"`
+	Monitored           *bool   `json:"monitored,omitempty"`
+	QualityProfileID    *int64  `json:"qualityProfileId,omitempty"`
+	MinimumAvailability *string `json:"minimumAvailability,omitempty"` // tba
```

Comment:
`MinimumAvailability` could probably be enum constants, but I do not know all the values.

Reply:
tba, announced, inCinemas, released, deleted

ref https://radarr.video/docs/api/#/MovieEditor/put_api_v3_movie_editor